### PR TITLE
General configs

### PR DESCRIPTION
**Pull request by FernandoBasso**
Add a few minor config stuff, a small section about running locally on README, and the favicon.

PS: will start to work on the actually content in the next few days.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
 source "https://rubygems.org"
+
+# Use this specific MRI ruby version.
+ruby "2.6.3"
```
Comment (lazyfrosch) [inline Gemfile]: Do we need to hardcode Ruby versions? Especially for github pages that sounds like a bad idea

**Review by lazyfrosch — CHANGES_REQUESTED** [review]
Everything else is fine!

File: Gemfile (hunk 1 above)
Comment (FernandoBasso) [reply Gemfile]: I guess you are right. Let's drop that line.

Comment (lazyfrosch) [reply Gemfile]: Don't forget lock :wink:

**Review by lazyfrosch — APPROVED** [review]
Thanks!

**Pull request merged**